db/products.js

```diff
@@ -0,0 +1,88 @@
+(function () {
+  const products = {
+    "products": [
+      {
+        name: "Sham Wow",
+        price: 40,
+        inventory: 1000,
+        id: 1
+      }
+    ]
+  };
+
+  const getProducts = () => {
+    return products;
+  };
+
+  const addProduct = (name, price, inventory) => {
+    if (!isNaN(parseInt(name)) || isNaN(parseInt(price)) || isNaN(parseInt(inventory))) {
+      return false;
+    }
+    let product = {};
+
+    product.id = products.products.length + 1;
+    product.name = name;
+    product.price = parseInt(price);
+    product.inventory = parseInt(inventory);
+
+    products.products.push(product);
+
+    return true;
+  };
+
+  const findProduct = (id) => {
+    let index = -1;
+
+    products.products.forEach(product => {
```

Comment:
The forEach has two other parameters (current, index, array), you could have used the index parameter there to set your index rather than doing another indexOf.